src/library_webgpu.js

```diff
@@ -2396,6 +2403,10 @@ var LibraryWebGPU = {
     return WebGPU.mgrSurface.create(context);
   },
 
+  wgpuInstanceHasWGSLLanguageFeature: (instance, featureEnumValue) => {
+    return navigator["gpu"]["wgslLanguageFeatures"].has(WebGPU.WGSLFeatureName[featureEnumValue]);
```

Comment:
We should probably return `false` if `navigator.gpu.wgslLanguageFeatures` doesn't exist. It was added a bit after the first version of the spec, and it doesn't look like Firefox has implemented it yet (didn't check Safari).

Reply:
Good catch! I tried it in Firefox and it was throwing an error.